lightning-invoice/src/utils.rs

```diff
@@ -218,6 +198,85 @@ where
 	}
 }
 
+/// Utility to filter channels for an invoice, and return the corresponding route hints to include
```

Comment:
nit: No need to say "Utility to". Simply say "Filters channels ... and returns ... ".